Source/WebKit/UIProcess/mac/WKTextAnimationManager.mm

```diff
@@ -124,10 +129,13 @@ - (void)addTextAnimationForAnimationID:(NSUUID *)uuid withData:(const WebCore::T
                 strongWebView->page().updateUnderlyingTextVisibilityForTextAnimationID(remainingID, false);
         }).get();
 
-        effect.get().completion = makeBlockPtr([weakWebView = WeakPtr<WebKit::WebViewImpl>(_webView), remainingID = data.unanimatedRangeUUID, uuid = RetainPtr(uuid), runMode = data.runMode] {
+        effect.get().completion = makeBlockPtr([weakSelf = WeakObjCPtr<WKTextAnimationManager>(self), weakWebView = WeakPtr<WebKit::WebViewImpl>(_webView), remainingID = data.unanimatedRangeUUID, uuid = RetainPtr(uuid), runMode = data.runMode] {
```

Comment:
```
weakWebView = WeakPtr<WebKit::WebViewImpl>(_webView)
```

is this actually needed?

Reply:
It could be `weakWebView = _webView`. This is how the existing code was though.